web/src/routes/auctions/[key]/+page.ts

```diff
@@ -1,4 +1,19 @@
-export function load({ params }) {
+import { getApiBaseUrl } from "$lib/utils";
+
+export async function load({ params }) {
     const { key } = params;
-    return { itemKey: key };
+
+    const auctionUrl = `${getApiBaseUrl()}/api/auctions/${key}`;
```

Comment:
getApiBaseUrl() already has the "/" at the end, so we should start with `api`, not `/api`!

Reply:
> Also, I am thinking whether we can just have this function in api.ts, because I think it belongs there...

You're right. Would you also move `getBaseUrl` so they're together? They're related to one another, but `getBaseUrl` is not related with the API...

Reply:
Hmm, good question, i don't know... getBaseUrl sounds more like it should stay in utils.ts. and so should getApiBaseUrl.

What I meant by api.ts was a function that retrieves an auction based on a key.